src/asset/entities.rs

```diff
@@ -122,6 +135,10 @@ impl PyLight {
     fn energy(&self) -> f32 {
         self.energy
     }
+
+    fn properties(&mut self) -> BTreeMap<String, String> {
```

Comment:
Looks like the rest of the properties() functions are not used in the python code, there is only Python implementations for importing the prop properties. Did you mean to implement the python code for the lights and unknown entities too?

Reply:
i will be honest, i have 0% knowledge of Rust, so I don't know how to answer that question. based on your commit those months ago for exposing unknown entities properties as custom properties, i tried looking for a pattern, then applied the same code to other entities, like lights and models.

however, whatever i DID do has allowed me to access entity properties on all lights, models, armatures, and empties. here is a light's properties from itemtest: `{'_cone': '75', '_constant_attn': '0', '_distance': '0', '_exponent': '1', '_fifty_percent_distance': '185', '_inner_cone': '45', '_light': '255 255 255 800', '_lightHDR': '-1 -1 -1 1', '_lightscaleHDR': '1', '_linear_attn': '0', '_quadratic_attn': '1', '_zero_percent_distance': '512', 'angles': '-90 90 0', 'origin': '-48.3398 -312.58 5.8928', 'pitch': '-90', 'style': '0'}`

Reply:
I meant that I expected to see changes in light.py and unknown_entity.py similar to the ones you have for prop.py. The Rust code looks good! Maybe these are missing from the PR?

Reply:
you're right, i'm only now understanding what you're talking about